project/Publish.scala

```diff
@@ -16,18 +16,21 @@ package org.apache.pekko
 import sbt._
 import sbt.Keys._
 import com.lightbend.sbt.publishrsync.PublishRsyncPlugin.autoImport.publishRsyncHost
-import xerial.sbt.Sonatype.autoImport.sonatypeProfileName
+import xerial.sbt.Sonatype.autoImport._
 
 object Publish extends AutoPlugin {
 
   override def trigger = allRequirements
 
+  private val apacheBaseRepo = "repository.apache.org"
+  private val apacheRepo = s"https://$apacheBaseRepo/content/repositories"
+
   override lazy val projectSettings = Seq(
-    publishTo := Some(pekkoPublishTo.value),
     publishRsyncHost := "akkarepo@gustav.akka.io",
     credentials ++= apacheNexusCredentials,
     organizationName := "Apache Software Foundation",
     organizationHomepage := Some(url("https://www.apache.org")),
+    sonatypeCredentialHost := apacheBaseRepo,
```

Comment:
This is the base repository which by default is `oss.sonatype.org` (and hence the reason why it was publishing to the wrong location before, the sbt-sonatype plugin sets `publishTo` to indirectly use `sonatypeCredentialHost` and other related sonatype settings).